src/standalone/Vegetation/PlantHydraulics.jl

```diff
@@ -370,6 +371,23 @@ function water_flux(z1::FT, z2::FT, ψ1::FT, ψ2::FT, K1::FT, K2::FT) where {FT}
     return flux # (m/s)
 end
 
+"""
+    root_distribution(z::FT, rooting_depth::FT)
+
+Computes value of rooting probabilty density function at `z`.
+
+The rooting probabilty density function is derived from the
+cumulative distribution function F(z) = 1 - beta^(100z), which is described
+by Equation 2.23 of
+Bonan, "Climate Change and Terrestrial Ecosystem Modeling", 2019 Cambridge University Press.
+This probability distribution function is equivalent to the derivative of the
+cumulative distribution function with respect to z,
+where `rooting_depth` replaces (-1)/(100ln(beta)) and z is expected to be negative.
+"""
```

Comment:
@kmdeck Is this clear and accurate? I was not sure if there is a citation convention to follow. 

In your comment you said to "say this is equivalent to this probability density function with coordinate z replacing depth," but the z coordinate in the CDF is still present in the PDF, just negated. 

Could you sketch out here the connection between rooting depth and beta?
E.g. give the CDF F(z) = 1-beta^(100z) and cite Equation 2.23 of Bonan, "Climate Change and Terrestrial Ecosystem Modeling", 2019 Cambridge University Press. And say this is equivalent to this probability density function with coordinate z replacing depth.

Reply:
I was making a distinction between "depth" z as Bonan uses it, and the coordinate z, what you have is clear! I would just check for typos ("probabilty" twice), replace beta with \beta (the symbol).

thank you!